src/main/java/com/todoist/pojo/Sanitizers.kt

```diff
@@ -3,10 +3,9 @@ package com.todoist.pojo
 import java.util.regex.Pattern
 
 object Sanitizers {
-    const val PROJECT_NAME_INVALID_CHARACTERS = "<>\"=#+"
+    const val PROJECT_NAME_INVALID_CHARACTERS = "[#\"]+"
```

Comment:
This is now a regex but those should be only characters, no?

Reply:
Oh, of course! It was an oversight from my part when copy-pasting from the spec. 🤦‍♂️